data/json/mapgen/house18.json

```diff
@@ -0,0 +1,147 @@
+[
+    {
+        "type": "mapgen",
+        "om_terrain": [
+            "house"
+        ],
+        "method": "json",
+        "weight": 300, "comment": "<--- low weight due to low randomness, remove when house mapgen pool is big",
```

Comment:
Yes. I just checked and actually the other houses are using comparable values so I'd leave it as is.